rancher/v1.2/en/quick-start-guide/index.md

```diff
@@ -31,12 +31,12 @@ If you are interested in trying one of our latest development builds which will
 All you need is one command to launch Rancher server. After launching the container, we'll tail the logs to see when the server is up and running.
 
 ```bash
-$ sudo docker run -d --restart=always -p 8080:8080 rancher/server
+$ sudo docker run -d --restart=always --name rancher-server -p 8080:8080 rancher/server
 # Tail the logs to show Rancher
-$ sudo docker logs -f containerid
+$ sudo docker logs -f rancher-server
 ```
 
-It will take a couple of minutes for Rancher server to start up. When the logs show `.... Startup Succeeded, Listening on port...`, Rancher UI is up and running.
+It will take a couple of minutes for Rancher server to start up. When the logs show `.... Startup Succeeded, Listening on port...`, Rancher UI is up and running. There could be more logs afterwards like `... Creating schema machine...`, so check that is hasn't already being outputted.
```

Comment:
OK, it's just that I waited about 10 min before realizing that the `listening on port...` was up in the logs. I thought it could help other newcomers.